src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -442,6 +442,14 @@ paths:
         required: true
```

Comment:
Re: `description`: Remember to mention that the OpenAPI-GUI takes care of encoding when experimenting.

Reply:
Description has been updated to include information on GUI handling encoding